kotak-website/src/home/App.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import './App.css';
+
+class Image extends Component{
```

Comment:
Lets split this component and all the other components in this file to their own files. 

Make a directory structure like 
```
/components/Image
/components/Bio
/components/Icon
```

If the component does not have a state, it can simply be a functional component. 
Here the image component does not have state and neither will it have state in future so it can simply be a functional component.